setup.cfg

```diff
@@ -17,13 +17,11 @@ classifiers =
 packages = find:
 install_requires =
     cartopy
-    dask-geopandas
```

Comment:
I think this package is still required for either image when working with a dask cluster, so it can be left as is.